addons/music/fnc_stopMusic.sqf

```diff
@@ -0,0 +1,27 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_stopMusic
+
+Description:
+    A function used to stop any music playing. Must have been started with CBA_fnc_playMusic, otherwise it is ignored. 
+    
+Parameters:
+    none
+
+Returns:
+    BOOL: true if music was stopped, false if already stopped
+    
+Example:
+    (begin example)
+        _bool = call CBA_fnc_stopMusic;
+    (end example)
+
+Author:
+    Fishy
+---------------------------------------------------------------------------- */
+#include "script_component.hpp"
+
+if (!([true] call CBA_fnc_getMusicPlaying)) exitWith {false};
```

Comment:
I think you could leave out this line.
It would result in the benefit, that this function stops music, which was not called via cba, too.